kong.conf.default

```diff
@@ -508,7 +508,7 @@
 #dns_no_sync = off               # If enabled, then upon a cache-miss every
                                  # request will trigger its own dns query.
                                  # When disabled multiple requests for the
-                                 # same name/type will be synchronised to a
+                                 # same name/type will be synchronized to a
```

Comment:
american vs english spelling. I'm not sure what our preferred style here is; I'd err on the side of leaving as-is.